data/categories.txt

```diff
@@ -10,9 +10,11 @@
 
 # Ships of these types can be purchased and also used as mission sources.
 category "ship"
-	"Transport"
 	"Light Freighter"
 	"Heavy Freighter"
+	"Transport"
+	"Space Liner"
+	"Utility"
```

Comment:
Instead of moving transport above freighter, I'd keep it where it is, add Space Liner above that, then have Utility above the Freighters.
```suggestion
	"Transport"
	"Space Liner"
	"Light Freighter"
	"Heavy Freighter"
	"Utility"
```